ios/FluentUI/Vnext/Notification/NotificationModifiers.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+public extension View {
```

Comment:
it shouldn't be a public extension of every view to add the ability to present notifications, right?

Reply:
can you elaborate? do you mean that we should have a protocol that views would have to conform to in order to be able to present notifications and instead extend off that protocol?

Reply:
I don't think we should have a specific protocol to limit the presentation of the NotificationView unless there is a requirement from that view.
Creating a public extension for View is how we ensure the NotificationView can be displayed on top of any SwiftUI view.

Reply:
seems like a unnecessary taxing on all swiftuiview to have this capability. when in reality they can just call the present notificatonview themselves? what's the overall size impact on this?

Reply:
@harrieshin creating a func in a View extension is the standard way of creating a SwiftUI modifier. We have to make an extension to the SwiftUI View to ensure that client apps can present the notification on top of the View they want.

The size impact is not significant as it's only about one new public func being introduced.